components/connectivity-adapter/internal/appregistry/service/converter.go

```diff
@@ -0,0 +1,24 @@
+package service
```

Comment:
What is the reasoning for having this sort of not implemented file?

Reply:
I created this empty file to show my idea for the project structure. Now we can split development: one person can write handlers, and one - conversion methods. I will define interface in the main handler file for the converter and keep this file as it is, if you don't mind. What do you think?

Reply:
OK, with the interface defined it makes more sense to me.